game/games/flappybird/Player.gd

```diff
@@ -0,0 +1,68 @@
+extends KinematicBody2D
+
+const UP = Vector2(0, -1)
+const FLAP_HEIGHT = 200  # flap time
+const MAXFALLSPEED = 200
+const GRAVITY = 10
+const END_MESSAGE := "You got %s points!"
+
+var timer = 0  # timer; spawning walls
+
+var motion = Vector2()
+var wall = preload("res://games/flappybird/WallNode.tscn")
+var score = 0
+
+
+func ready():
+	pass
+
+
+func _physics_process(delta):
+	timer += delta
+
+	if timer >= 1:
+		timer = 0
+		spawn_wall()
+
+	motion.y += GRAVITY
+	if motion.y > MAXFALLSPEED:
+		motion.y = MAXFALLSPEED
+
+	if Input.is_action_just_pressed("flap"):
+		if position.y > -80:
+			motion.y = -FLAP_HEIGHT
+
+	if position.y > 130:
+		GameManager.end_game(END_MESSAGE % score)
+		return
+
+	motion = move_and_slide(delta * 60 * motion, UP)
+
+	get_parent().get_parent().get_node("CanvasLayer/RichTextLabel").text = str(score)
+
+
+# spawn a new wall at the right side of the screen
+func spawn_wall():
+	var instance = wall.instance()
+	rand_seed(rand_range(0, 25))
```

Comment:
```suggestion
```
remove this

Reply:
thanks for the feedback.

i actually added line 47 because when running the game the normal rand_range created the same pattern over and over again. maybe it was another problem in the wall implementation but it was a quick fix.

i'll try to remove the line while fixing the problem

Reply:
You could do it like the testgame and use your own RandomNumberGenerator:
```gdscript
onready var _rng := RandomNumberGenerator.new()
```
and call `_rng.randomize()`

Reply:
i fixed it. thanks!